packages/amazonq/.changes/next-release/Bug Fix-b8c7f55f-2986-48a9-91e7-cf143faf51fc.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q Chat: When window gets focus, even though the autoFocus property is set to true, input field doesn't get focus"
```

avoid technical details in the changelogs
```suggestion
	"description": "Amazon Q Chat: Input field not getting focus when chat window opens"
```